bindgen/Cargo.toml

```diff
@@ -53,9 +53,9 @@ experimental = ["annotate-snippets"]
 
 ## The following features are for internal use and they shouldn't be used if
 ## you're not hacking on bindgen
-# Features used by `bindgen-cli` 
+# Features used by `bindgen-cli`
 __cli = []
-# Features used for CI testing 
```

Comment:
This is nice, but I'd rather have a different PR with this change to avoid churn